examples/convert_to_aster.py

```diff
@@ -0,0 +1,106 @@
+#Copyright (C) 2017-2018 COAL Developers
```

Comment:
@bdegley4789 there are a few major issues with the code in this file

1. The documentation does not mention that you need to have the file ```./ASCIIdata/ASCIIdata_splib07a/ChapterA_ArtificialMaterials/splib07a_Alizarin_crimson_(dk)_GDS780_ASDFRa_AREF.txt``` present in the current working directory in order for the conversion to work correctly... this would need to be corrected.
1. The logic is intrinsically linked to the man made compound [**Alizarin_crimson**](https://crustal.usgs.gov/speclab/data/HTMLmetadata/Alizarin_crimson_(dk)_GDS780_ASDFRa_AREF.html)... which is a paint pigment. There is absolutely no rational justification for us having an example of a conversion and spectral classification based upon a paint pigment as it has nothing to do with the COAL project. We are looking for (at a bare minimum) examples based on elements associated with COAL and surface mining activities e.g. [Renyolds_Tnl_Sludge](https://crustal.usgs.gov/speclab/data/HTMLmetadata/Renyolds_Tnl_Sludge_SM93-15_BECKa_AREF.html), [Renyolds_TnlSldgWet](https://crustal.usgs.gov/speclab/data/HTMLmetadata/Renyolds_TnlSldgWet_SM93-15w_BECKa_AREF.html) and [Schwertmannite](https://crustal.usgs.gov/speclab/data/HTMLmetadata/Schwertmannite_BZ93-1_BECKb_AREF.html). Make sure you ensure that the examples are related to the above. These proxy classes are extracted directly from [mining.py](https://github.com/capstone-coal/pycoal/blob/master/pycoal/mining.py#L23-L26). I've referenced these several times in the past so please take the guidance on board. Thanks.
1. The code in this file is a complete copy of the proposed code you've implemented in pycoal/mineral.py... why are you duplicating the code? You should just be using the ```SpectralToAsterConversion.convert(...)``` function you created in pycoal/mineral.py.